realgud/common/core.el

```diff
@@ -214,9 +214,10 @@ marginal icons is reset."
 	  (or (file-name-directory script-filename)
 	      default-directory "./"))
 	 (cmdproc-buffer-name
-	  (format "*%s %s shell*"
-		  (file-name-nondirectory debugger-name)
-		  (file-name-nondirectory script-filename)))
+       (replace-regexp-in-string "\s+" "\s"
```

Comment:
This looks wrong; it will incorrectly display file names that contain spaces, won't it?